scripts/update.py

```diff
@@ -224,8 +224,15 @@ def generate_repeater_df(args: Union[argparse.Namespace, SimpleNamespace]) -> pd
 
     # Initialize records missing state id fields to WA (53).
     if 'RepeaterBook State ID' not in df.columns:
-        df['RepeaterBook State ID'] = '53'
-    df['RepeaterBook State ID'] = df['RepeaterBook State ID'].fillna('53')
+        df['RepeaterBook State ID'] = numpy.NaN
+
+    def init_state_id(row):
+        if pd.notna(row['RepeaterBook State ID']):
+            return row['RepeaterBook State ID']
+        if pd.isna(row['RepeaterBook ID']):
+            return numpy.NaN
+        return '53'
+    df['RepeaterBook State ID'] = df.apply(init_state_id, axis=1)
```

Comment:
I don't think this is what we need permanently. Running this once will clean up / fix the repeaters.json file, but after that, we should be counting on the creation of new repeaters in this function to handle this correctly. I'll make a quick PR on that soon. 

Thanks !